lib/pbench/test/unit/server/test_state_tracker.py

```diff
@@ -17,7 +18,7 @@ class TestStateTracker:
     def test_state_enum(self):
         """ Test the States ENUM properties
         """
-        assert len(States.__members__) == 12
+        assert len(States.__members__) == 9
```

Comment:
Is this a useful and important thing to check?  (I.e., why does it matter that it is exactly `9`?...or, is this some clever way to remind us to update the unit tests when we add (or remove) a state?)

Reply:
> ...or, is this some clever way to remind us to update the unit tests when we add (or remove) a state?)

Yes. I suppose I could instead have enumerated all the expected ENUM values, but this was easy and seemed good enough.